docs/kit/operation-view/installation_guide.md

```diff
@@ -0,0 +1,116 @@
+    - `internationalDataSpaces.title`: the title of the IDS connector
+    - `internationalDataSpaces.maintainer`: the maintainer of the IDS connector
+    - `internationalDataSpaces.curator`: the curator profile of the IDS connector
+3. Setup secrets in [Hashicorp](https://github.com/eclipse-tractusx/tractusx-edc/blob/main/edc-extensions/hashicorp-vault/README.md) Vault, add the following secrets:
+    - `transfer-proxy-token-signer-public-key`: key `content`, key-value `-----BEGIN PUBLIC KEY-----...` (2048 bits+)
+    - `transfer-proxy-token-signer-private-key`: key `content`, key-value `-----BEGIN RSA PRIVATE KEY-----...` (2048 bits+)
+    - Connector certificate PEM as `daps-public-key`: key `content`, key-value `-----BEGIN CERTIFICATE-----...`
+    - private-key PEM as `daps-private-key`: key `content`, key-value `-----BEGIN PRIVATE KEY----- ...`
+    - `transfer-proxy-token-encryption-aes-key`: key `content`, aes-key as key-value
+4. Install the Tractus-X EDC (please replace the angle brackets data with the information from step 1):
+   ```bash
+   helm repo add tractusx-edc https://eclipse-tractusx.github.io/charts/dev
+   helm install edc tractusx-edc/tractusx-connector --version 0.3.3 --set vault.hashicorp.enabled=true --set postgresql.enabled=true --set vault.hashicorp.url=<YOUR URL> --set vault.hashicorp.token=<YOUR TOKEN> --set daps.clientId=<YOUR ID> --set controlplane.endpoints.management.authKey=<YOUR AUTHKEY> --set postgresql.username=<YOUR USERNAME> --set postgresql.password=<YOUR PW> --set postgresql.jdbcUrl=<YOUR JDBC URL> --set backendService.httpProxyTokenReceiverUrl=<YOUR URL> --set daps.url=<YOUR URL>
+   ```
+5. Check if the pods are running:
+   ```bash
+   kubectl get pods
+   ```
+6. To see logs of the pods, run the following command:
+   ```bash
+   kubectl logs -f <pod-name>
+   ```
+7. Following endpoints are now available in default setup:
+   - controlplane: 
+     - default: port `8080` `/api` 
+     - management: `8081` `/management`
+     - control: port `8083` `/control` 
+     - protocol: port `8084` `/api/v1/ids`
+     - observability: port `8085` `/observability`
+     - metrics: port `9090` `/metrics`
+   - dataplane:
+     - default: port `8080` `/api`
+     - public: port `8081` `/api/public`
+     - control: port `8083` `/api/dataplane/control`
+     - observability: port `8085` `/observability`
+     - metrics: port `9090` `/metrics`
+    
+8. How to continue?  
+You could register your connector in the Catena-X portal, for more information see [here](https://portal.int.demo.catena-x.net/documentation/?path=docs%2F02.+Technical+Integration).
+
+FAQ
+------------
+### How should the DAPS service URL be specified?  
+The URL of the DAPS depends on the dataspace you want to connect to. The best thing here is to contact a contact person and ask for the DAPS URL of the dataspace you want to join. The example URL will connect you to the Catena-X Int-Stage DAPS.
```

Comment:
this is the install guide for one particular dataspace, i.e. Catena-X, so we should list the DAPS's URL